src/components/VeggyHeader/HeaderCart/HeaderCart.jsx

```diff
@@ -1,14 +1,23 @@
-import CartImg from "../../../../public/img/cart.png"
-import './HeaderCart.css'
-import CartList from "./CartList/CartList.jsx";
 import useWidth from "../../../../hooks/useWindowResize.js";
+import CartList from "./CartList/CartList.jsx";
+import './HeaderCart.css';
+import CartImg from "../../../../public/img/cart.png";
+
 export default function HeaderCart (props) {
-    let productQuantity = props.addProduct.length;
-    let getEachProductSum = props.addProduct.map(product => product.sum);
+    const {products,
+        selectedProducts,
+        setSelectedProducts,
+        isModalActive,
+        setIsModalActive,
+        shakingCart,
+        setShakingCart
+    } = props;
+    const productQuantity = selectedProducts.length;
+    let getEachProductSum = selectedProducts.map(product => product.sum);
     let getAllProductSum  = getEachProductSum.reduce((sum, current) => sum + current, 0);
     let width = useWidth();
```

Comment:
```suggestion
    let width = useWIndowWidth();
```
В тебе так кастомний хук називається